src/System.IO.UnmanagedMemoryStream/ref/System.IO.UnmanagedMemoryStream.csproj

```diff
@@ -10,6 +10,9 @@
     <Compile Include="System.IO.UnmanagedMemoryStream.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="../../System.Runtime.InteropServices/ref/System.Runtime.InteropServices.csproj"/>
```

Comment:
Same here, basically every place where you added a temporary P2P reference just add a ToDo comment on it, as that is one of the ways we findstr the places we need to do it.